packages/babel-plugin/src/styled/__tests__/call-expression.test.ts

```diff
@@ -498,16 +498,14 @@ describe('styled object call expression', () => {
       export const BadgeSkeleton = styled.span({
         backgroundColor: ({ isLoading }) => isLoading ? colors.N20 : colors.N40,
         color: ({ loading: l }) => l ? colors.N50 : colors.N10,
-        borderColor: (propz) => propz.loading ? colors.N100 : colors.N200,
```

Comment:
Created https://github.com/atlassian-labs/compiled/issues/1167 to address the fact that props parameter must be called "props"